misc/nsis/x86.nsi

```diff
@@ -0,0 +1,66 @@
+Name "qutebrowser"
+OutFile "qutebrowser-x86.exe"
+
+Unicode true
+RequestExecutionLevel admin
+
+;Default installation folder
+InstallDir "$ProgramFiles\qutebrowser"
+  
+!include "MUI2.nsh"
+;!include "MultiUser.nsh"
+
+!define MUI_ABORTWARNING
+;!define MULTIUSER_MUI
+;!define MULTIUSER_INSTALLMODE_COMMANDLINE
+!define MUI_ICON "../../icons/qutebrowser.ico"
+!define MUI_UNICON "../../icons/qutebrowser.ico"
+
+!insertmacro MUI_PAGE_LICENSE "../../COPYING"
+!insertmacro MUI_PAGE_DIRECTORY
+!insertmacro MUI_PAGE_INSTFILES
+!insertmacro MUI_UNPAGE_CONFIRM
+!insertmacro MUI_UNPAGE_INSTFILES
+
+!insertmacro MUI_LANGUAGE "English"
+
+; depends on admin status
+;SetShellVarContext current
+
+
+Section "Install"
+
+  ; Uninstall old versions
+  ExecWait 'MsiExec.exe /quiet /qn /norestart /X{633F41F9-FE9B-42D1-9CC4-718CBD01EE11}'
+  ExecWait 'MsiExec.exe /quiet /qn /norestart /X{9331D947-AC86-4542-A755-A833429C6E69}'
+
+  SetOutPath "$INSTDIR"
+  
+  file /r "qutebrowser-x86\*.*"
```

Comment:
This should be `..\dist\qutebrowser-0.10.1-x86` (or `-x64`) to pick up what PyInstaller generated without moving it. Can `0.10.1` be a wildcard there too? If not, I guess it needs another define?